pulsar-checksum/pom.xml

```diff
@@ -53,29 +53,10 @@
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
     </dependency>
-
-    <dependency>
-      <groupId>com.scurrilous</groupId>
-      <artifactId>circe-api</artifactId>
-      <version>1.1-20140603.044855-1</version>
-    </dependency>
-
+    
     <dependency>
-      <groupId>com.scurrilous</groupId>
-      <artifactId>circe-impl</artifactId>
-      <version>1.1-20140603.044907-1</version>
-    </dependency>
-
-    <dependency>
-      <groupId>com.scurrilous</groupId>
-      <artifactId>circe-common</artifactId>
-      <version>1.1-20140603.044903-1</version>
-    </dependency>
-
-    <dependency>
-      <groupId>com.scurrilous</groupId>
-      <artifactId>circe-crc</artifactId>
-      <version>1.1-20140603.044857-1</version>
+      <groupId>com.googlecode.jmockit</groupId>
+      <artifactId>jmockit</artifactId>
```

Comment:
yes.. i missed one commit to push.. it's only required for test so, scope should be test. made the change.